cla_public/templates/macros/subform.html

```diff
@@ -20,7 +20,7 @@
         <li>{{ _('If you miss the call you’ll need to complete this form again.') }}</li>
       </ul>
     </div>
-    {{ Form.group(form.callback.contact_number, field_attrs={'class': 'm-large'}) }}
+    {{ Form.group(form.callback.contact_number, field_attrs={'class': 'm-large', 'autocomplete': 'tel', 'type': 'tel'}) }}
```

Comment:
I'm not fluent in the art of `field_attrs` 😞 
Could you explain how will this solve the problem, please?

Reply:
It's a dict of extra attributes to add (or merge with existing attributes values) on the rendered input element.

The `contact_number` input element is used to request a phone number to call users back. It will now be rendered with an `autocomplete="tel"` attribute, as per the guidance at https://design-system.service.gov.uk/patterns/telephone-numbers/

Additionally we set `type="tel"` for the extra nicety of optimising keyboard entry on mobile browsers.

This is the [quick win agreed on](https://mojdt.slack.com/archives/CFUESB43G/p1550848531052500?thread_ts=1550837360.034200&cid=CFUESB43G) for the ticket, before we consider introducing validation with `libphonenumber`.